utils/outputwriter/outputcontent.go

```diff
@@ -118,288 +157,719 @@ func footer(writer OutputWriter) string {
-	var contentBuilder strings.Builder
-	WriteContent(&contentBuilder,
-		writer.MarkAsTitle(sastTitle, 2),
-		writer.MarkInCenter(getJasDescriptionTable(severity, finding, writer)),
-		writer.MarkAsDetails("Full description", 3, fullDetails),
-	)
+func getSecretsDescriptionTable(severity, issueId, finding, status string, writer OutputWriter) string {
+	// Determine the issue applicable status
+	applicability := jasutils.Applicable.String()
+	if status == jasutils.Inactive.String() {
+		applicability = jasutils.NotApplicable.String()
+	}
+	columns := []string{"Severity"}
+	rowData := []string{writer.FormattedSeverity(severity, applicability)}
+	// Determine if issueId is provided
+	if issueId != "" {
+		columns = append(columns, "ID")
+		rowData = append(rowData, issueId)
+	}
+	columns = append(columns, "Finding")
+	rowData = append(rowData, finding)
+	// Determine if status is provided
+	if status != "" {
+		columns = append(columns, "Status")
+		rowData = append(rowData, status)
+	}
+	return NewMarkdownTable(columns...).AddRow(rowData...).Build()
+}
 
-	if len(codeFlows) > 0 {
-		WriteContent(&contentBuilder, writer.MarkAsDetails("Code Flows", 3, sastCodeFlowsReviewContent(codeFlows, writer)))
+func getSecretsFullDescription(issue formats.SourceCodeRow, violation bool, writer OutputWriter) string {
+	table := getBaseJasDetailsTable(issue.Watch, issue.Policies, issue.CWE, writer)
+	table.AddRow(MarkAsBold("Abbreviation:"), issue.RuleId)
+	return getJasFullDescription(issue, violation, table.Build(), writer)
+}
+
+// Utilities
+
+func getIssueType(violation bool) string {
+	if violation {
+		return "Violation"
 	}
-	return contentBuilder.String()
+	return "Vulnerability"
 }
 
-func sastCodeFlowsReviewContent(codeFlows [][]formats.Location, writer OutputWriter) string {
-	var contentBuilder strings.Builder
-	for _, flow := range codeFlows {
-		WriteContent(&contentBuilder, writer.MarkAsDetails("Vulnerable data flow analysis result", 4, sastDataFlowLocationsReviewContent(flow)))
+func getDirectDependenciesCellData(components []formats.ComponentRow) (dependencies CellData) {
+	if len(components) == 0 {
+		return NewCellData()
 	}
-	return contentBuilder.String()
+	for _, component := range components {
+		dependencies = append(dependencies, fmt.Sprintf("%s:%s", component.Name, component.Version))
+	}
+	return
 }
 
-func sastDataFlowLocationsReviewContent(flow []formats.Location) string {
+func getCveIdsCellData(cveRows []formats.CveRow, issueId string) (ids CellData) {
+	if len(cveRows) == 0 {
+		return NewCellData(issueId)
+	}
+	for _, cve := range cveRows {
+		ids = append(ids, cve.Id)
+	}
+	return
+}
+
+func getScaSecurityIssueDetailsContent(issues []formats.VulnerabilityOrViolationRow, violations bool, writer OutputWriter) (content []string) {
+	issuesWithDetails := getIssuesWithDetails(issues)
+	if len(issuesWithDetails) == 0 {
+		return
+	}
+	for _, issue := range issuesWithDetails {
+		if len(issues) == 1 {
```

Comment:
```suggestion
		if len(issuesWithDetails) == 1 {
```
Did you mean to refer this?